core/modules/filters/days.js

```diff
@@ -0,0 +1,57 @@
+/*\
+title: $:/core/modules/filters/days.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator that selects tiddlers with a specified date field within a specified date interval.
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+Math.sign = Math.sign || function(x) {
```

Comment:
Hi @nameanyone I think we should move this into utils.js. I suggest creating a function `$tw.utils.sign` with the same definition, rather than overwriting the built-in `Math.sign`.